satellite/console/consoleweb/server.go

```diff
@@ -193,10 +193,16 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 	var projectID *uuid.UUID
 	var since, before time.Time
 
-	tokenCookie, err := r.Cookie("tokenKey")
+	host, _, err := net.SplitHostPort(r.Host)
 	if err != nil {
 		server.log.Error("bucket usage report error", zap.Error(err))
+		server.serveError(w, r, http.StatusNotFound)
+		return
+	}
 
+	tokenCookie, err := r.Cookie(host + "_tokenKey")
+	if err != nil {
+		server.log.Error("bucket usage report error", zap.Error(err))
```

Comment:
We don't need to log as error the unauthorized request.
I see others below, but they should not do so.